src/shared/internal-calendar/standard/standard-calendar-day-view.tsx

```diff
@@ -74,17 +138,34 @@ export const StandardCalendarDayView = ({
     // =============================================================================
     const renderHeader = () => {
         return weeksOfTheMonth[0].map((day, index) => (
-            <HeaderCell key={`week-day-${index}`}>
+            <HeaderCell key={`week-day-${index}`} aria-hidden>
                 {dayjs(day).format("ddd")}
             </HeaderCell>
         ));
     };
 
     const renderDayCells = () => {
+        const firstDay =
+            minDate && calendarDate.isSame(dayjs(minDate), "month")
+                ? dayjs(minDate).date()
+                : 1;
         return weeksOfTheMonth.map((week, weekIndex) => {
             return (
-                <RowDayCell key={weekIndex} onMouseLeave={handleMouseLeaveCell}>
+                <RowDayCell
+                    role="row"
+                    aria-label={`Week ${weekIndex + 1}`}
+                    key={weekIndex}
+                    onMouseLeave={handleMouseLeaveCell}
+                    onBlur={handleMouseLeaveCell}
+                >
                     {week.map((day, dayIndex) => {
+                        const isDateToFocus = day.isSame(dateToFocus, "day");
+
+                        // First day of the month, should always be in tab order
+                        const isFirstDayOfMonth =
+                            day.date() === firstDay &&
+                            day.month() === calendarDate.month();
```

Comment:
this seems inconsistent with month/year as there could be 2 different tab spots in 1 grid (the first day of month + selected date). though I might have missed a requirement

if there's only 1 tab spot, it would be cleaner to keep the logic for the focusable date in one place (i.e. the `dateToFocus` constant that you introduced) and we can simplify to `tabIndex={day.isSame(dateToFocus, "day") ? 0 : 1}`